ECLIPs/ECLIP-2.md

```diff
@@ -0,0 +1,50 @@
+# Add OpenRPC Service Discovery
+
+    ECLIP: 2
+    Title: Add OpenRPC Service Discovery
```

Comment:
...to multi-geth specifically? or to multigeth/geth/parity

Reply:
trying to keep it pretty generic so it could be proposed as say a BIP

Reply:
or for anything that has a JSONRPC server, eh?

Reply:
yea exactly